.isort.cfg

```diff
@@ -1,15 +1,24 @@
 [settings]
 multi_line_output=3
-include_trailing_comma=True
+include_trailing_comma=true
 force_grid_wrap=0
 use_parentheses=True
 line_length=88
-ensure_newline_before_comments=True
-known_third_party=omegaconf,ray,pytest,typing_extensions
-known_first_party=hydra
+ensure_newline_before_comments=true
+known_libs=pkg_resources,setuptools
+sections=
+    FUTURE
+    STDLIB
+    LIBS
+    THIRDPARTY
+    FIRSTPARTY
+    LOCALFOLDER
+default_section=THIRDPARTY
+known_first_party=hydra,hydra_app,configen,example
 skip=
     plugins
+    ,__init__.py
     ,.nox
     ,hydra/grammar/gen
     ,tools/configen/example/gen
-    ,tools/configen/tests/test_modules/expected
+    ,tools/configen/tests/test_modules/expected
```

Comment:
can you add a trailing newline?

Reply:
I'll add it for this and in the `noxfile.py`

Reply:
Does this work? Trailing newlines:
![Screen Shot 2020-09-17 at 12 06 40 PM](https://user-images.githubusercontent.com/14094630/93497553-8968db00-f8de-11ea-8b61-ad41be517152.png)
![Screen Shot 2020-09-17 at 12 07 23 PM](https://user-images.githubusercontent.com/14094630/93497555-8b329e80-f8de-11ea-8fe1-5fc5657233fb.png)

Reply:
yes